### ISSUE TITLE
Do not show solutions that are not there

### ISSUE THREAD

**Issue by erikvullings** [DRIVER-EU/test-bed-admin#9]
E.g. your FRQ COP tool is not available, so do not show it in the list.

**Comment by tobritzh** [comment]
The admin tool HMI shows that what it gets from the service, and in the current "example" configuration we added one Solution (LifeX COP). This can be easily changed in the config file. For the Trial, it has to be predefined which solutions are part of it.

**Comment by JoaquinMB-Freq** [comment]
It is in the list because it is set in the config file
Hence, again it's not a bug

**Comment by erikvullings** [comment]
Agreed, but I don't want an admin tool user to first edit the config files in order to define his trial environment. Related to auto discovery of services, etc.. 

It may actually be useful if, in the configuration message, we add a property so adapter users can describe their own service, and, optionally, provide a link to it. Than we don't need to define this information in the config files. 
What do you think?
